packages/client/lib/functions/ProofUtil.ts

```diff
@@ -1,8 +1,15 @@
+import {
+  BAD_PARAMS,
+  JWS_NOT_VALID,
+  Jwt,
+  JWTPayload,
+  ProofOfPossession,
+  ProofOfPossessionCallbacks,
+  ProofType,
+} from '@sphereon/openid4vci-common/lib';
```

Comment:
This is referencing a dir in the package, suggesting the exports are not in order. Should not have to include /lib

Reply:
exports were fine. Intellij auto-refactor upone move added the folder.
Removed the lib part from path.